docs/content/deploy.md

```diff
@@ -28,98 +24,58 @@ To start a local deployment, either:
 
 This starts a local version of the SpatialOS Runtime on your development machine together with all server-workers specified in the launch configuration you used.
 
+> **It’s done when:** You see the following message in the terminal: `SpatialOS ready. Access the Inspector at http://localhost:21000/inspector`
+
 To start your client-workers, from a terminal window, in any directory, run `spatial local worker launch <YourExternalWorker> <YourLaunchConfig>`, replacing the `<example-content>` with names relevant to your game. Use the [Inspector](https://docs.improbable.io/reference/latest/shared/operate/inspector) to look at the current state of your game world.
 
 ## Cloud deployment
 
+To start a cloud deployment follow the section.
```

Comment:
"...follow _this_ section."

or

"Follow this section to start a cloud deployment"